### DISCUSSION TITLE
Community release of v9

### DISCUSSION THREAD

**Discussion by MartinRobins** [PrismLibrary/Prism#3004] [General]
With .NET8 now having been released, are you able to provide any guidance as to when there will be a non-pre-release version of Prism9 for the community (as in non paid)?

Thanks.

**Comment by dansiegel** [comment]
Prism 9.0 will NOT ship with the MIT license as we have in the past. We are committed to ensuring that Prism is a sustainable project which is why we have introduced the new license to help make Prism a sustainable project. 

We have tried to be as generous as we can with the Community License being 100% Free, and requiring no actual registration. If you earn less than $1,000,000 USD per year and are developing an app personally you qualify for the Community License. If you are developing for your company and your company either earns more than $1,000,000 USD in gross receipts per year, or has ever received $3,000,000 USD or more in startup capital then you would be required to purchase a License. We have kept the license fees intentionally low and reasonable for people.

For Consultants... You may have to have multiple licenses as the license conditions apply to the owner of the IP which is rarely if ever the Consultant. The license conditions apply to your clients as the IP owner. If you have a client that would require a Commercial License then they must purchase a license for any In House developers, Consultants or others who need to build the app.

We are not doing a separate release. When we release Prism 9 (which will be very soon...) it will be available publicly on NuGet.org. For those with the Commercial Plus license they MAY (or may not) get additional bug fixes which are backported, in addition to the latest CI packages and some packages that are not available publicly including the Prism.Magician, Prism.Plugin.Essentials, Prism.Plugin.Popups.Maui, and additional container support for Microsoft.Extensions.DependencyInjection and Grace Ioc.

**Reply by MartinRobins** [reply]
@dansiegel  Thanks for the clarification. I look forward to v9 and in the meantime am speaking to the necessary people as to whether or not we qualify for the community license or whether we will need to look at the other options.

**Reply by dansiegel** [reply]
Honestly even if you qualify for the Community License, you may want to consider the Commercial Plus license because it's actually very reasonably priced and we're including a bunch of extra stuff that just will make your life easier...

**Reply by jumoone** [reply]
`we're including a bunch of extra stuff that just will make your life easier`

I don't see details as to what the tools actually do. This would be an easier sell to my management if we knew what the extra stuff did.

Is there a way to purchase the license via PO? I only see options for credit card, cash app, and bank transfer.

**Reply by brianlagunas** [reply]
We will be updating the website with a list of all the "stuff" you get with the Commercial Plus license soon.

Yes, you can purchase with a PO, but we only accept Credit Card, Bank Transfer/ACH for payments.
